### PR TITLE
asynchronously subscribe to user connections

### PR DESCRIPTION
**Pull request by efim-poberezkin**

### CODE REVIEW

File: src/Simplex/Chat/Mobile.hs [hunk 1]
```diff
@@ -49,7 +49,8 @@ mobileChatOpts =
   ChatOpts
     { dbFilePrefix = "simplex_v1", -- two database files will be created: simplex_v1_chat.db and simplex_v1_agent.db
       smpServers = defaultSMPServers,
-      logging = False
+      logConnections = False,
```
Comment (efim-poberezkin) [inline src/Simplex/Chat/Mobile.hs:52]: `logSubscriptions` might've been more descriptive, but it's okay to keep as is
```suggestion
      logSubscriptions = False,
```

File: src/Simplex/Chat/Options.hs [hunk 2]
```diff
@@ -55,9 +56,14 @@ chatOpts appDir =
           <> value defaultSMPServers
       )
     <*> switch
-      ( long "log"
+      ( long "connections"
+          <> short 'c'
+          <> help "Log every contact and group connection on start"
```
Comment (efim-poberezkin) [inline src/Simplex/Chat/Options.hs:61]: same here
```suggestion
      ( long "subscriptions"
          <> short 'c'
          <> help "Log every contact and group subscription on start"
```

File: src/Simplex/Chat/Options.hs [hunk 3]
```diff
@@ -21,7 +21,8 @@ import System.FilePath (combine)
 data ChatOpts = ChatOpts
   { dbFilePrefix :: String,
     smpServers :: NonEmpty SMPServer,
-    logging :: Bool
+    logConnections :: Bool,
```
Comment (efim-poberezkin) [inline src/Simplex/Chat/Options.hs:24]:
```suggestion
    logSubscriptions :: Bool,
```

File: src/Simplex/Chat/Options.hs (hunk 2 above)
Comment (epoberezkin) [reply src/Simplex/Chat/Options.hs:61]: subscription is our internal term, the current message in UI is "contact is connected"

**Pull request merged**